diagnostic.md

```diff
@@ -27,7 +27,9 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+h1 {
+  font-size: $base-font-size + 30%;
```

Comment:
This should be `font-size: $base-font-size * 1.3;` since `$base-font-size` might not be defined as a percentage.